app/lib/actions/katello/content_view_version/incremental_update.rb

```diff
@@ -166,8 +166,9 @@ def generate_description(version, content)
             end
             humanized_lines << ''
           end
-          version.description = humanized_lines.join("\n")
-          version.save!
+          content_view = version.content_view
+          content_view.description = humanized_lines.join("\n")
+          version.save! && content_view.save!
```

Comment:
My vote would be to append.  I think both pieces of information are important.  

The purpose of this was to display to the user what was actually updated in the Content view.  Maybe in the future we store links to the actual units that were modified so we can better display them in a list (rather than a text blob).